BLAG.Common/Models/QuestionVideo.cs

```diff
@@ -1,13 +1,10 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.ComponentModel.DataAnnotations;
-using System.Text;
 
 namespace BLAG.Common.Models
 {
     public class QuestionVideo : QuestionBase
     {
-        [Required]
-        public byte[] Video { get; set; }
+        // String will hold the ID of where the video file is saved in LiteDB
+        public String Video { get; set; }
```

Comment:
You should use the keyword `string` instead of the type name `String`. If resharper didn't warn you let me know.